packages/cli/src/lib/transform.ts

```
@@ -1,34 +1,61 @@
-import { execFileSync } from "child_process";
+import { execFileSync, spawnSync } from "child_process";
 import debug from "debug";
 import path from "path";
 import { TransformOptions } from "./transform-options";
 import { fileURLToPath } from "url";
+import * as fs from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)